modules/data-entry/src/main/frontend/src/dataHomepage/ExportButton.jsx

```diff
@@ -0,0 +1,395 @@
+                />
+              </Grid>
+            </Grid>
+
+            <Grid container alignItems='center' direction="row" className={classes.container}>
+              <Grid item xs={4}><Typography variant="subtitle2">Data format:</Typography></Grid>
+              <Grid item xs={8}>
+                <RadioGroup
+                  row
+                  defaultValue="values"
+                  aria-label="data"
+                  name="data"
+                  value={hasLabeles}
+                  onChange={(event) => setHasLabeles(event.target.value === "true")}
+                >
+                  <FormControlLabel value={true} control={<Radio />} label="Labels" />
+                  <FormControlLabel value={false} control={<Radio />} label="Values" />
+                </RadioGroup>
+              </Grid>
+            </Grid>
+
+            <Grid container alignItems='center' direction="row" className={classes.container}>
+              <Grid item xs={4}><Typography variant="subtitle2">Column selection mode:</Typography></Grid>
+              <Grid item xs={8}>
+                <RadioGroup
+                  row
+                  defaultValue="isinclude"
+                  aria-label="isinclude"
+                  name="isinclude"
+                  value={isInclude}
+                  onChange={(event) => setIsInclude(event.target.value === "true")}
+                >
+                  <FormControlLabel value={true} control={<Radio />} label="Include" />
+                  <FormControlLabel value={false} control={<Radio />} label="Exclude" />
+                </RadioGroup>
+              </Grid>
+            </Grid>
+
+            <Grid container alignItems='center' direction="row" className={classes.container}>
+              <Grid item xs={4}>
+                <Typography variant="subtitle2">{isInclude ? "Columns to include:" : "Columns to exclude:"}</Typography>
+              </Grid>
+              <Grid item xs={8}>
+
+                {/* List the entered values */}
+                { entities?.filter(v => questionIds.includes(v.path)).map((value, index) =>
+                  <Grid container
+                    key={`${value.name}-${index}`}
+                    direction="row"
+                    justifyContent="space-between"
+                    alignItems="stretch"
+                    className={classes.valueEntry}
+                  >
+                    <Grid item xs={9}>
+                      <Avatar
+                        style={{backgroundColor: entitySpecs[value.type].avatarColor || "black"}}
+                        className={classes.avatar}
+                        sx={{ width: 30, height: 30 }}
+                      >
+                        { entitySpecs[value.type].avatar ? <Icon>{entitySpecs[value.type].avatar}</Icon> : value.type?.charAt(0) }
+                      </Avatar>
+                      <ListItemText primary={value.name} secondary={value.text || value.label} />
```

Comment:
`label` is no longer necessary here, we're saving the section label in `text`
```suggestion
                      <ListItemText primary={value.name} secondary={value.text} />
```